cypress/tests/data/20-CreateContext.spec.js

```diff
@@ -113,23 +123,19 @@ describe('Data suite tests', function() {
 
 		// Submit the form with required fields missing.
 		cy.get('div[id=contact').find('button').contains('Save').click();
-		cy.get('div[id="contact-contactName-error"]').contains('This field is required.');
-		cy.get('div[id="contact-contactEmail-error"]').contains('This field is required.');
 		cy.get('div[id="contact-supportName-error"]').contains('This field is required.');
 		cy.get('div[id="contact-supportEmail-error"]').contains('This field is required.');
 
-		cy.get('input[name=contactName]').type('Ramiro Vaca', {delay: 0});
+		// cy.get('input[name=contactName]').type('Ramiro Vaca', {delay: 0});
```

Comment:
Just a reminder about a commented line here and on line 138

Reply:
removed those commented lines .